src/phpDocumentor/Descriptor/Builder/PhpParser/FunctionAssembler.php

```diff
@@ -40,11 +40,11 @@ public function __construct(ArgumentAssembler $argumentAssembler)
      *
      * @param Function_ $data
      *
-     * @return FunctionDescriptor
+     * @return Function_
      */
     public function create($data)
     {
-        $functionDescriptor = new FunctionDescriptor();
+        $functionDescriptor = new Function_();
```

Comment:
I take it that the `Function_` element should still be refactored since it does not receive any constructor arguments (such as the FQCN)?